src/app/ReadHandler.h

```diff
@@ -358,8 +384,13 @@ class ReadHandler : public Messaging::ExchangeDelegate
     /**
      * Called internally to signal the completion of all work on this objecta and signal to a registered callback that it's
      * safe to release this object.
+     *
+     *  @param    keepPersisted             Keep the subscription persisted in storage for later resumption. Ignored if
```

Comment:
use an enum not a bool. Boolean arguments are hard to understand if not obvious. `setEnabled(true)` is obvious. `Close(true)` is not (does true mean "really close" and false "do not close"?).

Reply:
Done.